src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/InterArrival.java

```diff
@@ -124,12 +132,40 @@ private boolean belongsToBurst(long arrivalTimeMs, long timestamp)
      * {@code arrivalTimeDeltaMs} is the computed arrival-time delta,
      * {@code packetSizeDelta} is the computed size delta.
      * @return
+     *
+     * @Note: We have two {@Link computeDeltas}.
+     * One with a valid {@Link systemTimeMs} according to webrtc
+     * implementation as of June 12,2017 and a previous one
+     * with a default systemTimeMs (-1L). the later may be removed or
+     * deprecated.
      */
     public boolean computeDeltas(
             long timestamp,
             long arrivalTimeMs,
+            long systemTimeMs,
+            int packetSize,
+            long[] deltas){
+
+        return solveDeltas(timestamp,arrivalTimeMs,
+                packetSize,deltas,systemTimeMs);
+    }
+
+    public boolean computeDeltas(
+            long timestamp,
+            long arrivalTimeMs,
+            int packetSize,
+            long[] deltas){
+
+                return solveDeltas(timestamp,arrivalTimeMs,
+                        packetSize,deltas,-1L);
+    }
+
+    public boolean solveDeltas(
```

Comment:
Does the name solveDeltas come from webrtc.org, or is it something that we added?
It seems unnecessary, as we can have the 4-argument computeDeltas call the 5-argument computeDeltas directly.